components/centos/centos-openshift-setup/Vagrantfile

```diff
@@ -115,8 +115,8 @@ Vagrant.configure(2) do |config|
     echo "Configured cluster admin user: admin, Password: admin"
     echo
     echo "To use OpenShift CLI, run:"
-    echo "$ vagrant ssh"
-    echo "$ oc login"
+    echo "vagrant service-manager install-cli openshift"
```

Comment:
Yes. Why I proposed that because, when instructions are visible to the command line, the users will do that as first thing. He might already have OpenShift binary which is already set and can be used. Then, he will decide whether to install it or not.

Probably better wordings will help here.